randovania/games/prime1/logic_database/Chozo Ruins.txt

```diff
@@ -2027,27 +2034,51 @@ Extra - aabb: [397.9558, -8.934963, 4.7323303, 483.05167, 117.17189, 59.302334]
   > Pickup (Missile Expansion)
       All of the following:
           Any of the following:
+              # Deal with Water
               After Flaahgra or Poisoned Water Damage ≥ 57
               Gravity Suit and Poisoned Water Damage ≥ 25
           Any of the following:
+              # Get up
+              # Bomb Jump
               Gravity Suit and Bomb Jump (Beginner) and Poisoned Water Damage ≥ 15 and Can Use Bombs
-              # https://youtu.be/Jla-gYSErGc
+              # Gravless: https://youtu.be/Jla-gYSErGc
               No Gravity Suit and Slope Jump (Advanced) and Gravityless Underwater Movement (Advanced) and Enabled Allow Dangerous Gravity Suit Logic
               All of the following:
+                  # Jump up with Space Jump
                   Space Jump Boots
                   Any of the following:
                       Gravity Suit
                       Slope Jump (Beginner) and Gravityless Underwater Movement (Beginner)
               All of the following:
-                  # https://youtu.be/HDYJkv3MUVI
+                  # Wallboost: https://youtu.be/HDYJkv3MUVI
                   Gravity Suit and Wall Boost (Beginner) and Can Use Boost Ball
                   Any of the following:
                       Movement (Intermediate) and Poisoned Water Damage ≥ 65
                       Movement (Expert) and Poisoned Water Damage ≥ 20
   > Event - Open gate
-      Scan Visor
+      All of the following:
+          Scan Visor
+          Any of the following:
+              # Dealing with Vines/Blastcaps
+              Shoot Any Beam
+              All of the following:
+                  # Tank Blastcap and get back by jumping through water
+                  Combat (Intermediate) and Normal Damage ≥ 13
```

Comment:
I argue that this is not combat:
- The creature does not force this interaction on the player. The player's involvement in this situation is completely within their control
- Our description for combat is:

> If this trick is enabled, the player may be expected to defeat enemies and bosses with fewer items and less health. On Beginner or higher, Charge Beam may be locked behind many of the game's mandatory fights.

Even if you considered this ambient creature an enemy, there is no way to take less damage in this situation by being skillful. Destroying them with your toes is just a minimum health tax you must pay to solve this puzzle.

Reply:
> having this trickless feels wrong

you got me there,

how about combat beginner

Reply:
I was considering combat beginner first, but there can be some huge variance on the amount of damage you take from the blastcaps.

But i suppose that's what the damage leniency setting is for.